src/components/GameSwitchboard.js

```diff
@@ -30,6 +57,20 @@ class GameSwitchboard extends React.Component {
             )
         }
         
+        if(gamestate === 'enterScore') {
+            return (
+                <div>
+                    <h3> Enter Information </h3>
+                    <label htmlFor="scoreName"> Name: </label>
+                    <input type="text" id="scoreName" name="scoreName"/>
+                    <button type="button" onClick={this.submitScoreInfo}> 
```

Comment:
Here's the docs on how they suggest doing this with react: https://reactjs.org/docs/forms.html